prototypes/entity.lua

```diff
@@ -59,16 +60,115 @@ data:extend({
 		{
 			shadow =
 			{
-				red = {2.01, 0.6},
-				green = {2.52, 0.6}
+				red = {2.52, 0.65},
+				green = {2.01, 0.65}
 			},
 			wire =
 			{
-				red = {1.71, 0.3},
-				green = {2.22, 0.3}
+				red = {2.22, 0.32},
+				green = {1.71, 0.32}
 			}
 		},
 	},
+	-- small warehouse
+	{
+		type = "container",
+		name = "warehouse-basic-small",
+		icon = "__Warehousing__/graphics/icons/warehouse-basic-small.png",
+		icon_size = 32,
+		flags = {"placeable-neutral", "placeable-player", "player-creation"},
+		minable = {mining_time = 2, result = "warehouse-basic-small"},
+		max_health = 450,
+		corpse = "big-remnants",
+		dying_explosion = "medium-explosion",
+		open_sound = { filename = "__base__/sound/machine-open.ogg", volume = 0.85 },
+		close_sound = { filename = "__base__/sound/machine-close.ogg", volume = 0.75 },
+		vehicle_impact_sound = { filename = "__base__/sound/car-metal-impact.ogg", volume = 0.65 },
+		resistances =
+		{
+			{
+				type = "fire",
+				percent = 90
+			}
+		},
+		collision_box = {{-2.4, -2.4}, {2.4, 2.4}},
```

Comment:
The collision box is meant to be smaller than the selection box. See the `-basic` elements—their collision boxes are 0.3 units smaller than their selection boxes.

Reply:
Discussion will continue on another of these similar changes.